source/index.html

```diff
@@ -2,11 +2,160 @@
   </head>
   <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
-
+    <h1 class="visually hidden">CAT ENERGY</h1>
+    <header class="main__header">
+      <a class="header__logo logo__link" href="#">
+        <img class="logo-link__img" width="33" height="37" alt="Логотип с котом." src="../source/img/icon/logo_mobile.svg">
+      </a>
+      <a class="header__logo-main logo-main__link" href="#">
+        <img class="logo-main__img" width="101" height="18" alt="Логотип магазина" src="../source/img/icon/logo_main.svg">
+      </a>
+      <button class="main-header__button-burger" type="button"></button>
+      <div class="main-header__popover popover__menu">
+        <button class="menu__button-close" type="button">
+          <span class="visually-hidden">Закрыть.</span>
+        </button>
+        <div class="menu__content">
+          <ul class="menu__content-list">
+            <li class="menu__content-item">
+              <a class="menu__content-link">Главная</a>
+            </li>
+            <li class="menu__content-item">
+              <a class="menu__content-link" href="catalog.html">Каталог продукции</a>
+            </li>
+            <li class="menu__content-item">
+              <a class="menu__content-link" href="form.html">Подбор программы</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </header>
+    <main>
+      <section class="section__promo">
+        <h2 class="promo__title">Функциональное питание для котов</h2>
+        <p class="promo__text">Занялся собой? Займись котом!</p>
+        <img class="promo__img" width="280" height="270" alt="Стакан корма с курицей." src="../source/img/catalog/banner.png">
+        <a class="promo__link" href="#">Подобрать программу</a>
+        <ul class="promo__list">
+          <li class="promo__item">
+            <h3 class="promo__item-subtitle">Похудение</h3>
+            <p class="promo__item-text">Ваш кот весит больше собаки и почти утратил способность лазить по деревьям? Пора на диету! Cat Energy Slim поможет вашему питомцу сбросить лишний вес.</p>
+            <div class="promo__wrapper-link wrapper__link">
+              <a class="promo__link" href="#">Каталог slim</a>
+            </div>
+          </li>
+          <li class="promo__item">
+            <h3 class="promo__item-subtitle">Набор массы</h3>
+            <p class="promo__item-text">Заработать авторитет среди дворовых котов и даже собак? Серия Cat Energy Pro поможет вашему коту нарастить необходимые мышцы!</p>
+            <div class="promo__wrapper-link wrapper__link">
+              <a class="promo__link" href="#">Каталог pro</a>
+            </div>
+          </li>
+        </ul>
+        <div class="promo__advantages">
+          <h3 class="advantages__title">Как это работает</h3>
+          <ul class="advantages__list">
```

Comment:
На других размерах есть номер, поэтому предположим что порядок важен
```suggestion
          <ol class="advantages__list">
```